xsl/pageTools/frameWork.xsl

```diff
@@ -168,81 +169,82 @@
             <!-- These stylesheets are used on nearly everypage , so define them here to make them global -->
          <link href="{$css-loc}frameWork.css"      media="all" type="text/css" rel="stylesheet"/>
          <link href="{$css-loc}header.css"         media="all" type="text/css" rel="stylesheet"/>
-         <link href="{$css-loc}quickSearchBar.css" media="all" type="text/css" rel="stylesheet"/>
```

Comment:
These whitespace changes shouldn't have been included.  Looks like some indentation was lost in places.  Since the indentation of a lot of these files is wildly inconsistent throughout, this can go through but should be watched for in future by checking `git diff` before each commit to see if anything unexpected has changed.